src/package.xml

```diff
@@ -304,6 +304,8 @@
         <members>UTIL_Currency_TEST</members>
         <members>UTIL_CustomSettingsFacade</members>
         <members>UTIL_CustomSettings_API</members>
+        <members>UTIL_DMLService</members>
```

Comment:
These 2 files shouldn't be in here. I think they're just out of order in the file, but they should be there already so this would duplicate them.

Reply:
Ok. I just removed them and pushed the change.